webpage/webstreaming.py

```diff
@@ -41,14 +43,38 @@ def get_post_select_mode_data():
 
     @app.route('/data')
     def spectro_data():
+        global t0
         yAxe = det.get_last_spectrum()
-        return jsonify({'results':yAxe})
+        xAxe = det.get_wavelengths()
+        t = time()
+        # print("TIME DATA:",round(t-t0,3))
+        t0 = t
+        return jsonify({'yAxe':yAxe,'xAxe':xAxe})
+
+    @app.route('/web_config')
+    def set_web_config():
+        global t1
+        yAxe = det.get_last_spectrum()
+        xAxe = det.get_wavelengths()
+        t = time()
+        # print("TIME CONFIG:",round(t-t1,3))
+        t1 = t
+        integration_time = det.integration_time
+        auto_en = bool("automatic" == det.operation_mode)
+        return jsonify({'integration_time': integration_time,'auto_en':auto_en})
 
     @app.route("/save_spectrum", methods=["GET","POST"])
     def save_spectrum():
         det._save_spectrum(det.location,det.get_last_spectrum())
         return "Ok"
 
+    @app.route("/stop_graph", methods=["GET","POST"])
+    def stop_graph():
+        state = request.form['state'] == 'true'
```

Comment:
I don't like this, if we could it would be better to set the state to boolean, although I understand it can be very complicated through js/html